common.mk

```diff
@@ -0,0 +1,19 @@
+PROJECT_ROOT := $(shell git rev-parse --show-toplevel)
+PATH := $(PATH):$(PROJECT_ROOT)/scripts
+
+SHELL := env PATH=$(PATH) /bin/bash
```

Comment:
this will fail if there is a space in any path in $PATH, due to lack of escaping.  Suggest:
> SHELL := env PATH='$(PATH)' /bin/bash